.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  # It is also possible to trigger via the API:
+  # https://docs.github.com/en/rest/actions/workflows#create-a-workflow-dispatch-event
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "The tag of the version that is being deployed"
+        required: true
+        type: string
+
+name: Make a GitHub release with a deployment changelog
```

Comment:
Can we move `name` to the first line? I'm not sure if there's a best practice, but it just makes it easier to read and more organized